### DISCUSSION TITLE
Brouter Farben

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#187] [Q&A] [answered]
Kann bitte jemand sagen was die Farben in der Weltkarte in Brouter bedeuten. Grün sind die Abschnitte zum auswählen. 
Blau die heruntergeladenen Bereiche ?
Gelb die aktivierten ?
Heißt das man wählt immer nur die Bereiche aus die man gerade braucht ?
Gibt es eine Beschränkung wieviel Bereiche man auf einmal wählen kann ?
Sorry für die blöden Fragen aber die offroad Planung ist eine super Sache welche ich in Zukunft vermehrt einsetzen möchte.
Kann der Speicherverbrauch für die 4 Bereiche stimmen.
<img src="https://github.com/devemux86/cruiser/assets/124886589/a340a708-ecf0-4210-84c8-c540b35a4b33" width="50%"/>

**Comment by devemux86** [comment, marked as answer]
@1maxi2

BRouter colors can be found here:
Die BRouter-Farben finden Sie hier:

- https://github.com/devemux86/cruiser/discussions/137#discussioncomment-6029728

**Comment by devemux86** [comment]
@1maxi2 

BRouter uses all downloaded regions in the route calculation.

BRouter verwendet alle heruntergeladenen Regionen bei der Routenberechnung.

**Reply by 1maxi2** [reply]
Danke für die Info !

**Comment by devemux86** [comment]
> offroad Planung ist eine super Sache welche ich in Zukunft vermehrt einsetzen möchte.

:+1:

**Reply by 0709wiwiwi** [reply]
= "offline" Planung.

To even better support the standard routed "onroad" trajects that are partly mixed with some [offroad](https://github.com/devemux86/cruiser/discussions/79#discussioncomment-5535928) trajectory please add a thumbs up in that tread !

**Reply by 1maxi2** [reply]
> > Offroad Planung ist eine super Sache, die ich in Zukunft vermehrt einsetzen möchte.
> 
> 👍

Meinte natürlich " offline" Planung.
Bin aber auch ein Fan von offroad Strecken.
Bin übrigens gerade in Kreta auf Urlaub 😊
 Bin vom Norden der Insel in den Süden gefahren. Leider nur mit dem Auto. Auch hier hat Cruiser gute Dienste geleistet ! 
Allerdings gibt es Richtung Süden traumhafte Strecken und Gegenden so das es fast egal ist welche Straßen man fährt !
